### ISSUE TITLE
Update READme in Repo

### ISSUE THREAD

**Issue by alejmedinajr** [alejmedinajr/AI-Detector#11]
The ReadME in the repo needs to be updated to reflect information about the project. This issue can also be reopened to update the ReadME throughout the completion of the project.

**Comment by alejmedinajr** [comment]
I think the README file is updated to as much as it can be for how much is known about the project. As more progress and features are added/completed, then I will update the README to reflect this. For now, I think it is appropriate to leave it at this.

**Comment by alejmedinajr** [comment]
Going to update the readme so it gives some more instruction on how to properly store API keys in the config file.

**Comment by alejmedinajr** [comment]
The readme is now updated to reflect the addition of the `config.py` file that should be created and used to store private information such as API Keys.

**Comment by alejmedinajr** [comment]
@Caleb-Highsmith and I met for thirty minutes and discussed what changes should be made to the readme to better help the project in terms of new users using our components. We also talked about requirements for the project and how we would go about them (also planning a way to incorporate a requirements file to automate this process). We also made recommended changes to the abstract. All of this can be found in today's meeting notes (located in meeting notes folder of the shared google drive).
